### PR TITLE
add liquify

### PR DESCRIPTION
**Pull request by zzvara**
Awesome Contribution Checklist:

- [x] I have read, and re-read the [Contributing Guidelines](https://github.com/chartjs/awesome/blob/master/CONTRIBUTING.md)
- [x] I have searched to ensure the suggested item doesn't exist on this list
- [x] This PR contains only one item

### Please Describe Your Addition

Add link to [https://github.com/sakos95/liquify]. Liquify is for fast, multi-threaded visualization of stream data with ChartJS & Angular.

Live charts

### CODE REVIEW

**Review by sakos95 — DISMISSED** [review]
Liquify is a cool library, that can be really useful to visualize high-velocity data streams.

**Comment by benmccann** [comment]
Cool! Just FYI, Chart.js 3.0 will have better support for multi-threading, which you might be interested in: https://www.chartjs.org/docs/next/general/performance#render-chartjs-in-a-web-worker-chrome-only

File: README.md [hunk 1]
```diff
@@ -74,3 +74,4 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 - [nova-chartjs](https://github.com/coroo/nova-chartjs) - Laravel Nova
 - [quickchart](https://github.com/typpo/quickchart) - Web API for static charts
 - [wicked-charts](https://github.com/adessoAG/wicked-charts) - Java
+- [liquify](https://github.com/sakos95/liquify) - Fast, multi-threaded visualization of stream data with Angular
```
Comment (kurkle) [inline README.md]: "Additions should be added in alphabetical order to the relevant section."

Comment (zzvara) [reply README.md]: @kurkle Thanks for pointing out and sorry for slipping on that detail.

**Review by etimberg — APPROVED** [review]

**Review by sakos95 — APPROVED** [review]

**Review by kurkle — APPROVED** [review]

**Review by benmccann — APPROVED** [review]

**Pull request merged**